packages/api/src/services/actions.ts

```diff
@@ -79,6 +79,17 @@ export function Handlers(config: Config, appState: Dependencies): Actions {
       // convert this to tuple to save bytes.
       return results.map(({ price, timestamp }) => [timestamp, price]);
     },
+    async getEmpStats(address: string, currency: CurrencySymbol = "usd") {
+      assert(address, "requires address");
+      assert(currency, "requires currency");
+      assert(stats[currency], "No stats for currency: " + currency);
+      return stats[currency].latest.get(address);
+    },
+    async listEmpStats(currency: CurrencySymbol = "usd") {
+      assert(currency, "requires currency");
+      assert(stats[currency], "No stats for currency: " + currency);
+      return stats[currency].latest.values();
+    },
   };
 
   // list all available actions
```

Comment:
would it be useful to have a function that updates the state and then returns? i.e I want the _latest_ stat for an emp.

Reply:
stats are a unique case currently where all the calculations come from our internal state, and doing an update would be pretty cheap and require no third party calls.  Its not really a safe assumption to make in general though (an unprivileged user could spam third party calls), so I would propose that we try to maintain the latest state in the table itself through another more controlled process.  Currently this is done through polling every X seconds. 

At the moment its not really necessary to be super realtime, but if we were dealing with bots, or something that required the performance, i would change updating this table to be event based, and listen for changes on values which could change the stats object, this way we queries would always be up to date without needing to force an update on query.